elasticsearch/probe/readiness.sh

```diff
@@ -45,4 +45,4 @@ function check_for_init_complete() {
 }
 
 check_if_ready "/" "Elasticsearch node is not ready to accept HTTP requests yet"
-check_for_init_complete
+check_for_init_complete || cat ${HOME}/init_failures || :
```

Comment:
I don't think theres a need for `|| :` at the end. If `check_for_init_complete` failed at all that file should exist